docs/system-administration/DNS/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Self Hosting a Caching / Authoritative Internal DNS Sever.
+summary: A simple introduction on DNS.
+date: 2020-11-26
+authors:
+  - Adam Spann
+---
```

Comment:
I'm not sure this frontmatter is actually shown on the site?

Reply:
I think you are correct. But its part of the standard mkDocs.
I think the title: is used. but I will need to double check something.

Reply:
Also having this meta data now means that it can be used in the future. It also might be a good standard template.
I don't know but I would hope that the summary might be used when and if the site generates its sitemap.

Reply:
The frontmatter does work in the case of debian-stretch.md. Not sure why it does work for index.md.

Reply:
Not sure how to document this. But. I will do my best.
Title is used to the left pane
<img width="228" alt="Left Pane" src="https://user-images.githubusercontent.com/6175103/101104388-5812c800-360e-11eb-873e-7a75c9d65a5b.png">

Within index.md
If I remove the "# Self Hosting a Caching / Authoritative Internal DNS Sever." There is no Table of contents on the right pane.
<img width="1330" alt="Right-pane-no-header" src="https://user-images.githubusercontent.com/6175103/101104649-eedf8480-360e-11eb-81ce-3976110f4ddc.png">

If I keep the header
<img width="1328" alt="Right-pane-wth-header" src="https://user-images.githubusercontent.com/6175103/101104661-f737bf80-360e-11eb-8ddd-85c5934bcd63.png">